### PR TITLE
fix #341 by adding more tests for Ior

### LINKED ISSUE #341: Better test coverage for Ior
starting work on this

### PR DESCRIPTION
**Pull request by woparry**

### CODE REVIEW

**Comment by ceedubs** [comment]
Wow that code coverage update is really handy. Thanks @InTheNow and @woparry!

**Comment by non** [comment]
:+1:

File: tests/src/test/scala/cats/tests/IorTests.scala [hunk 1]
```diff
@@ -10,4 +14,114 @@ class IorTests extends CatsSuite {
 
   checkAll("Ior[String, Int] with Option", TraverseTests[String Ior ?].traverse[Int, Int, Int, Int, Option, Option])
   checkAll("Traverse[String Ior ?]", SerializableTests.serializable(Traverse[String Ior ?]))
+
+  implicit val arbitraryIor: Arbitrary[Ior[Int, String]] = Arbitrary {
```
Comment (rossabaker) [inline tests/src/test/scala/cats/tests/IorTests.scala]: Is there a reason we can't use `cats.laws.discipline.arbitrary.iorArbitrary`?

**Comment by rossabaker** [comment]
:+1:

**Pull request merged**